app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from __future__ import annotations
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.druid import Druid
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+
+
+def calculate_team_total_rating(
+        player_list:
+        list[Druid | ElfRanger | DwarfWarrior | DwarfBlacksmith]
```

Comment:
```suggestion
        list[Player]
```
You can use only Player because all of these classes inherit from Player.